app/views/users/_publishing_from_rss.html.erb

```diff
@@ -37,6 +50,7 @@
   <div class="field">
     <label></label>
     <%= f.hidden_field :tab, value: @tab %>
-    <%= f.submit "SUBMIT", class: "cta" %>
+    <% button_text = @user.feed_url.present? ? "UPDATE" : "SUBMIT" %>
+    <%= f.submit button_text, class: "cta" %>
```

Comment:
I understand why you added this but I'd keep the text as `SUBMIT` for consistency with all the other settings pages.

Reply:
I actually asked about this and @maestromac said it was fine.